GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.TranscriptModel+GVA.swift

```diff
@@ -0,0 +1,126 @@
+import Foundation
+
+private extension String {
+    static let gvaOptionUrlTargetModal = "modal"
+    static let gvaOptionUrlTargetSelf = "self"
+}
+
+extension SecureConversations.TranscriptModel {
+    func quickReplyOption(_ gvaOption: GvaOption) -> QuickReplyButtonCell.Props {
+        let action = Cmd { [weak self] in
+            self?.gvaOptionAction(for: gvaOption)()
+            self?.action?(.quickReplyPropsUpdated(.hidden))
+        }
+        return .init(
+            title: gvaOption.text,
+            action: action
+        )
+    }
+
+    func gvaOptionAction(for option: GvaOption) -> Cmd {
+        // If `option.destinationPdBroadcastEvent` is specified,
+        // this is broadcast event button, which is not supported
+        // on mobile. So an alert should be shown.
+        // If option contains `url`, then it's URL Button,
+        // otherwise it's Postback Button and option should be sent
+        // to the server as `SingleChoiceOption`
+        if option.destinationPdBroadcastEvent != nil {
+            return broadcastEventButtonAction()
+        } else if let urlString = option.url,
+                  let url = URL(string: urlString) {
+            return urlButtonAction(url: url, urlTarget: option.urlTarget)
+        } else {
+            return postbackButtonAction(for: option)
+        }
```

Comment:
Each case has `return`, would not it be more readable to have this?
```swift
        if option.destinationPdBroadcastEvent != nil {
            return broadcastEventButtonAction()
        }

        if let urlString = option.url, let url = URL(string: urlString) {
            return urlButtonAction(url: url, urlTarget: option.urlTarget)
        }

        return postbackButtonAction(for: option)
    }
```

Reply:
Fixed 82f63a05d94ad40c9f6b7661d4af245705902b76